### PR TITLE
docs: update instructions for JavaScript and Python on how to use gRPC

### LINKED ISSUE #4233: Rendezvous of RPC that terminate in new SubscribeHtlcEvents
### Background
Hi, I've used this new https://api.lightning.community/?python#subscribehtlcevents call, (after fixing a bug in the documentation... ) 
This is the code that I used
```python
import codecs, grpc, os
import router_pb2 as lnd, router_pb2_grpc as routerrpc
import time
import pickle


macaroon = codecs.encode(open('/home/admin/.lnd/data/chain/bitcoin/mainnet/admin.macaroon', 'rb').read(), 'hex')
os.environ['GRPC_SSL_CIPHER_SUITES'] = 'HIGH+ECDSA'
cert = open('/home/admin/.lnd/tls.cert', 'rb').read()
ssl_creds = grpc.ssl_channel_credentials(cert)
channel = grpc.secure_channel('localhost:10009', ssl_creds)
stub = routerrpc.RouterStub(channel)
request = lnd.SubscribeHtlcEventsRequest()
print("Running")                                                                                                                                                                                               
responses = []                                                                                                                                                                                                 
response_id = 0                                                                                                                                                                                                
for response in stub.SubscribeHtlcEvents(request, metadata=[('macaroon', macaroon)]):                                                                                                                          
    print('res_id = ' + str(response_id))                                                                                                                                                                      
    print(response)

```

But after a while (at res_id 19) I got this error:
```
_Rendezvous: <_Rendezvous of RPC that terminated with:                                                                                                                                      
        status = StatusCode.UNKNOWN           
        details = "unknown failure detail type: <nil>"                                                                                                                                       
        debug_error_string = "{"created":"@1588220504.572403636","description":"Error received from peer ipv4:127.0.0.1:10009","file":"src/core/lib/surface/call.cc","file_line":1052,"grpc_m
essage":"unknown failure detail type: <nil>","grpc_status":2}"
```

### Your environment

lnd version 0.10.0-beta commit=v0.10.0-beta
Linux rock64 4.4.132-1075-rockchip-ayufan-ga83beded8524 #1 SMP Thu Jul 26 08:22:22 UTC 2018 aarch64 GNU/Linux

Thanks

### PR DESCRIPTION
**Pull request by guggero**
To reflect the recent changes in the API docs, the instructions/HOW-TOs for JavaScript and Python needed some polishing.

Fixes #4233

### CODE REVIEW

File: docs/grpc/javascript.md [hunk 1]
```diff
@@ -18,49 +19,57 @@ at least somewhere reachable by your Javascript code).
 The `rpc.proto` file is [located in the `lnrpc` directory of the `lnd`
 sources](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/rpc.proto).
 
-In order for the auto-generated code to compile successfully, you'll need to
-comment out the following line:
-
-```
-//import "google/api/annotations.proto";
```
Comment (wpaulino) [inline docs/grpc/javascript.md:25]: Do we know why this isn't necessary anymore?

File: docs/grpc/javascript.md [hunk 2]
```diff
@@ -18,49 +19,57 @@ at least somewhere reachable by your Javascript code).
...
 
 ```js
-> lightning.getInfo({}, function(err, response) {
-  	console.log('GetInfo:', response);
-  });
+lightning.getInfo({}, function(err, response) {
+  if (err) {
+    console.log('Error: ' + err);
+  }
+  onsole.log('GetInfo:', response);
```
Comment (wpaulino) [inline docs/grpc/javascript.md:71]:
```suggestion
  console.log('GetInfo:', response);
```

File: docs/grpc/python.md [hunk 3]
```diff
@@ -3,46 +3,61 @@
...
-```
-(lnd)$ python -m grpc_tools.protoc --proto_path=googleapis:. --python_out=. --grpc_python_out=. rpc.proto
-```
+    ```
+    (lnd)$ curl -o rpc.proto -s https://raw.githubusercontent.com/lightningnetwork/lnd/master/lnrpc/rpc.proto
+    ```
+6. Compile the proto file
+    ```
+    (lnd)$ python -m grpc_tools.protoc --proto_path=googleapis:. --python_out=. --grpc_python_out=. rpc.proto
+    ```
```
Comment (carlaKC) [inline docs/grpc/python.md:39]: nit: separate formatting changes into another commit?

File: docs/grpc/python.md [hunk 4]
```diff
@@ -189,3 +204,9 @@ Python](https://developers.google.com/protocol-buffers/docs/pythontutorial).
 Additionally, [this official gRPC
 resource](http://www.grpc.io/docs/tutorials/basic/python.html) provides more
 details around how to drive `gRPC` from Python.
+
+## API documentation
+
+There is an [online API documentation](https://api.lightning.community?python)
```
Comment (carlaKC) [inline docs/grpc/python.md:210]: 🎉

File: docs/grpc/javascript.md [hunk 5]
```diff
@@ -18,49 +19,57 @@ at least somewhere reachable by your Javascript code).
...
+const packageDefinition = protoLoader.loadSync('rpc.proto', loaderOptions);
+
 //  Lnd cert is at ~/.lnd/tls.cert on Linux and
 //  ~/Library/Application Support/Lnd/tls.cert on Mac
-var lndCert = fs.readFileSync("~/.lnd/tls.cert");
-var credentials = grpc.credentials.createSsl(lndCert);
-var lnrpcDescriptor = grpc.load("rpc.proto");
-var lnrpc = lnrpcDescriptor.lnrpc;
-var lightning = new lnrpc.Lightning('localhost:10009', credentials);
+let lndCert = fs.readFileSync("~/.lnd/tls.cert");
```
Comment (carlaKC) [inline docs/grpc/javascript.md:50]: split let -> var out into it's own commit as well?

File: docs/grpc/javascript.md (hunk 1 above)
Comment (guggero) [reply docs/grpc/javascript.md:25]: Yes, the `proto-loader` library just ignores these annotations, even if it cannot find the referenced proto file. I've verified it is no longer needed by running the example code on a fresh system.
